nautobot_netbox_importer/tests/fixtures/3.0/summary.txt

```diff
@@ -331,7 +331,7 @@ name (DATA) => value_importer => name (CharField)
 * dcim.devicebaytemplate => dcim.devicebaytemplate *************************************************
 id (CUSTOM) => uid_from_data => id (UUIDField)
 * dcim.devicerole => extras.role *******************************************************************
-color (DATA) (CUSTOM) => value_importer => color (CharField)
+color (DATA) => value_importer => color (CharField)
```

Comment:
Unrelated change?

Reply:
This file is generated, don't want to change it manually.

Noticed this color field flapping. Want to fix it in a separate PR, not a real issue now.